### PR TITLE
stop using python script for cppunit tests

### PR DESCRIPTION
**Pull request by arnej27959**
@geirst please review and merge
@vekterli @baldersheim FYI

### CODE REVIEW

File: storage/src/tests/CMakeLists.txt [hunk 1]
```diff
@@ -32,5 +32,6 @@ vespa_add_executable(storage_testrunner_app TEST
 
 vespa_add_test(
     NAME storage_testrunner_app
+    NO_VALGRIND
```
Comment (geirst) [inline storage/src/tests/CMakeLists.txt]: Was this intentional? Do we really want to run without valgrind? @baldersheim

**Review by geirst — COMMENTED** [review]
Overall looks good. One comment regarding valgrind we need to decide first

File: storage/src/tests/CMakeLists.txt (hunk 1 above)
Comment (arnej27959) [reply storage/src/tests/CMakeLists.txt]: it was intentional - that test suite took a really long time with valgrind, moving to gtest will fix this also i guess...

Comment (vekterli) [reply storage/src/tests/CMakeLists.txt]: I am not comfortable running storage tests without valgrind. How much of an impact does this really have compared to the rest of the build process?

Comment (geirst) [reply storage/src/tests/CMakeLists.txt]: Then I suggest we run storage tests with valgrind as before. @arnej27959 can you fix?

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**